### PR TITLE
Avatar and Text - design tokens

### PR DESCRIPTION
**Pull request by M-i-k-e-l**
## Description
Avatar - design tokens

WOAUILIB-2596

## Changelog
Avatar - design tokens

### CODE REVIEW

File: src/components/avatar/index.tsx [hunk 1]
```diff
@@ -407,7 +407,7 @@ function createStyles(props: AvatarProps) {
       lineHeight: undefined
     },
     ribbon: {
-      backgroundColor: Colors.primary,
+      backgroundColor: Colors.$backgroundPrimaryLight,
```
Comment (lidord-wix) [inline src/components/avatar/index.tsx:410]: I think it should be `$backgroundPrimaryHeavy`, no?

**Review by lidord-wix — COMMENTED** [review]
Small comment, also please add `$textDefault` to the screen titles so it'll change in dark mode

File: src/components/avatar/index.tsx (hunk 1 above)
Comment (M-i-k-e-l) [reply src/components/avatar/index.tsx]: Indeed

**Comment by lidord-wix** [comment]
> Small comment, also please add `$textDefault` to the screen titles so it'll change in dark mode

@M-i-k-e-l I think you missed the comment about the screen titles :)

**Comment by M-i-k-e-l** [comment]
> @M-i-k-e-l I think you missed the comment about the screen titles :)

Fixed

**Review by lidord-wix — APPROVED** [review]

**Pull request merged**